### DISCUSSION TITLE
Hide Secrets and Config Map on UI

### DISCUSSION THREAD

**Discussion by gorakhgaurav** [argoproj/argo-cd#15120] [Q&A]
Hi Team,

I have deployed ArgoCD on my Red Hat OpenShift cluster. I have given read-only access to the dev team and admin to the DevOps team. 

As per management requirements, they want to hide secrets and config map visibility to the dev team. I want to know how can I only hide secrets and config map from the dev team and have read access on other resources?

**Comment by gaeljw** [comment]
I don't think that's possible currently. 

Secrets values are hidden in ArgoCD UI anyway.

ConfigMap is not supposed to contain sensitive values so I would challenge a bit this need.

**Reply by gorakhgaurav** [reply]
Yes config map shouldn't be used to keep sensitive info but is it possible to hide them on UI or stop read access as well for dev team ?

**Reply by jannfis** [reply]
As @gaeljw said, it's not possible to limit access to specific resource types.
